guides/source/active_record_basics.md

```diff
@@ -288,18 +288,12 @@ user = User.find_by(name: 'David')
 user.update(name: 'Dave')
 ```
 
-This is most useful when updating several attributes at once. If, on the other
-hand, you'd like to update several records in bulk, you may find the
-`update_all` class method useful:
+This is most useful when updating several attributes at once. 
 
-```ruby
-User.update_all "max_login_attempts = 3, must_change_password = 'true'"
-```
-
-This is the same as if you wrote:
+If you'd like to update several records in bulk without callbacks or validations, you can update the database directly using `update_all`
```

Comment:
We try to wrap documentation to [80/100 characters](https://github.com/rails/rails/pull/37885#discussion_r400931085)
```suggestion
If you'd like to update several records in bulk without callbacks or
validations, you can update the database directly using `update_all`
```

Reply:
Thanks, done in new squashed commit.